app/views/application/_home_page_listings.html.erb

```diff
@@ -11,6 +12,7 @@
     </div>
     <div class="box col-lg-3">
       <%= image_tag "ex_new_york.jpg" %>
+			<a href="/listings?city=New York">
```

Comment:
Really like the simple a tag, this will be easy to switch up in the future if need be! :+1: